reference/user_defined_functions.rst

```diff
@@ -457,6 +457,89 @@ Example
                                  540007
   (1 row)
 
+citus_relation_size
```

Comment:
Added these to the "Metadata / Configuration Information" reference since size is metadata of a sort.